modules/stochastic_tools/doc/content/source/distributions/FDistribution.md

```diff
@@ -0,0 +1,21 @@
+# F-Distribution
+
+!syntax description /Distributions/FDistribution
+
+## Overview
+
+This object defines a [F-distribution](https://en.wikipedia.org/wiki/F-distribution) with two degree of freedom parameters: [!param](/Distributions/FDistribution/df1) ($d_1$) and [!param](/Distributions/FDistribution/df2) ($d_2$). The probability density function and cumulative distribution function are defined in [eq:f_pdf] and [eq:f_cdf], respectively.
+
+!equation id=eq:f_pdf
+f(x; d_1, d_2) = \frac{1}{\Beta\left(\frac{d_1}{2},\frac{d_2}{2}\right)} \left(\frac{d_1}{d_2}\right)^{\frac{d_1}{2}} x^{\frac{d_1}{2}-1} \left(1+\frac{d_1}{d_2}x\right)^{-\frac{d_1-d_2}{2}},
```

Comment:
This is complicated to read in line. I will wait for the docs to compile so I can read on the website :)